crates/openvino/src/blob.rs

```diff
@@ -87,7 +87,7 @@ impl Blob {
     /// # Panics
     ///
     /// Panics if the returned OpenVINO size will not fit in `usize`.
-    pub fn byte_len(&mut self) -> Result<usize> {
+    pub fn byte_len(&self) -> Result<usize> {
```

Comment:
I believe these `&mut self` to `&self` changes make sense in a Rust context but I propagated the `mut` due to `ie_blob_byte_size`'s signature (e.g.), which takes a `*mut ie_blob_t`. I wasn't sure if this was just some bindgen over-reaction or if indeed there was some possible underlying mutation. Looking at `ie_blob.h` now, I see that both `size` and `byteSize` are declared `const` member functions so this should be safe.